docs/docs/cmd/spo/folder/folder-retentionlabel-ensure.md

```diff
@@ -0,0 +1,47 @@
+# spo folder retentionlabel ensure
+
+Apply a retention label to a folder
+
+## Usage
+
+```sh
+m365 spo folder retentionlabel ensure [options]
+```
+
+## Options
+
+`-u, --webUrl <webUrl>`
+: URL of the site where the retentionlabel from a file to apply is located
+
+`--folderUrl [folderUrl]`
+: The server-relative URL of the folder that should be labelled. Specify either `folderUrl` or `folderId` but not both.
+
+`i, --folderId [folderId]`
+: The UniqueId (GUID) of the folder that should be labelled. Specify either `folderUrl` or `folderId` but not both.
+
+`--name <name>`
+: Name of the retention label to apply to the folder.
+
+--8<-- "docs/cmd/_global.md"
+
+## Remarks
+
+Adding a retentionlabel is only supported on subfolders. Adding a retentionlabel to a rootfolder is currently not supported by by this command.
+
+## Examples
+
+Applies a retention label to a folder based on the label name and the folderUrl
+
+```sh
```

Comment:
Could you also mention the `spo listitem retentionlabel remove` command? Just to show and help users to find what they need, that they can achieve the same with another command. Something like: 'You can also use the...'

Reply:
Added it, but i don't really understand why we should mention the listitem remove command from the folder ensure command? Feels like it's not really the same (listitem vs folder, remove vs ensure)